tests/unit/mesh/test_ncmesh.cpp

```diff
@@ -0,0 +1,239 @@
+// Copyright (c) 2010-2020, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#include "mfem.hpp"
+#include "catch.hpp"
+
+namespace mfem
+{
+
+constexpr double EPS = 1.e-12;
+
+// Test case: Verify that a conforming mesh yields the same diagonal with PA
```

Comment:
This is picky but: what you actually test is that the *norms* of the diagonals are the same. That's fine for a simple unit test, but the comments should reflect what it actually tests.

Reply:
Thanks, @barker29! I made a minor to change to clarify this point.